preferences.py

```diff
@@ -147,8 +147,9 @@ class LOOKINGGLASS_PT_preferences(AddonPreferences):
 
 	# logger level
 	logger_level: bpy.props.EnumProperty(
-									items = [('0', 'Debug messages', 'All messages are written to the log file. This is for detailed debugging and extended bug reports.'),
```

Comment:
FYI this change (deleting the `.` at the end) is due to blender automatically adding a period at the end of tooltip descriptions, so it would appear as a double dot.

Reply:
Yes, thanks. I do this mistake regularly although I am aware of it. Maybe I learn it some day ... ;)

Reply:
I thought I was going insane just now, until I realized: When you hover over the dropdown *before* selecting anything, no `.` is added to the popup description, but once you expand the dropdown and then hover over one of *those* options, blender then adds a `.` to the end of the description. So blender itself seems a tad inconsistent it seems. Minor things anwyays!

Reply:
Ah, interesting. Maybe that is also the origin of my confusion with that. Anyway, as you said it's a minor thing.